lib/msal-common/src/authority/Authority.ts

```diff
@@ -19,6 +19,8 @@ import { CloudInstanceDiscoveryResponse, isCloudInstanceDiscoveryResponse } from
 import { CloudDiscoveryMetadata } from "./CloudDiscoveryMetadata";
 import { RegionDiscovery } from "./RegionDiscovery";
 import { RegionDiscoveryMetadata } from "./RegionDiscoveryMetadata";
+import { AzureCloudOptions } from "../config/ClientConfiguration";
+import { AzureCloudInstance } from "..";
```

Comment:
This will create a circular dependency. Import from the actual file instead

Reply:
Typo, correcting